src/Microsoft.AspNetCore.Razor.VSCode/src/extension.ts

```diff
@@ -105,6 +105,7 @@ export async function activate(context: ExtensionContext, languageServerDir: str
 
         context.subscriptions.push(languageServerClient, onStartRegistration, onStopRegistration, logger);
     } catch (error) {
+        logger.logError('Failed when activating Razor VSCode.', error);
```

Comment:
When moving the projected document providers to the vscode adapter I realized we weren't reporting activation errors to the Razor log. Added this bit as a bonus.